src/ducks/categorization/services.js

```diff
@@ -213,24 +216,54 @@ export const localModel = async (classifierOptions, transactions) => {
       'No classifier, impossible to categorize transactions'
     )
     return
+  } else {
+    return classifier
   }
+}
+
+export const pctOfTokensInVoc = (tokens, vocabularyArray) => {
+  const n_tokens = tokens.length
+  const intersection = tokens.filter(t => -1 !== vocabularyArray.indexOf(t))
+  return intersection.length / n_tokens
+}
+
+export const localModel = async (classifierOptions, transactions) => {
+  const classifier = await createLocalModel(classifierOptions)
```

Comment:
Here, we should check if `classifier` really exists and exit early if it doesn't.